lib/foreman_ansible_core/ansible_runner_task_launcher.rb

```diff
@@ -98,6 +108,12 @@ def prepare_directory_structure
         end
       end
 
+      def log_event(description, event)
+        # TODO: replace this ugly code with block variant once https://github.com/Dynflow/dynflow/pull/323
+        # arrives in production
+        logger.debug("[foreman_ansible] - handling event #{description}: #{JSON.pretty_generate(event)}") if logger.level <= ::Logger::DEBUG
```

Comment:
Since this is doing some additional processing while generating the data (`JSON.pretty_generate`), I wanted to execute it only in specific log level. Unfortunately, it's not possible to use `logger.debug { }` before https://github.com/Dynflow/dynflow/pull/323 gets merged, the condition with `if` is just a workaround for the time beaing.